kodata/events/toc-review.rotation

```diff
@@ -0,0 +1,19 @@
+#@ title: ToC Working Group Update
```

Comment:
can you move this to `%root/events` and symlink it back to `kodata/events`? I just did this for the rotations.

Reply:
Done.